lib/bokken_web/controllers/fallback_controller.ex

```diff
@@ -34,4 +34,25 @@ defmodule BokkenWeb.FallbackController do
     |> put_view(BokkenWeb.ErrorView)
     |> render(:"401")
   end
+
+  def call(conn, {:error, :ninja_already_enrolled}) do
+    conn
+    |> put_status(:forbidden)
+    |> put_view(BokkenWeb.ErrorView)
+    |> render(:"403")
+  end
+
+  def call(conn, {:error, :enrollments_closed}) do
+    conn
+    |> put_status(:forbidden)
+    |> put_view(BokkenWeb.ErrorView)
+    |> render(:"403")
+  end
+
+  def call(conn, {:error, :not_ninja_guardian}) do
+    conn
+    |> put_status(:forbidden)
+    |> put_view(BokkenWeb.ErrorView)
+    |> render(:"403")
+  end
```

Comment:
In this case, it is okay to all be 403 because they should never happen :bulb: 

```suggestion
  def call(conn, {:error, reason}) when reason in [:ninja_already_enrolled, :enrollments_closed, :not_ninja_guardian] do
    conn
    |> put_status(:forbidden)
    |> put_view(BokkenWeb.ErrorView)
    |> render(:"403")
  end
```

Reply:
You should move those reasons to Bokken.Guards.is_403 :muscle: